pkg/proto/aksnodeconfig/v1/helper.go

```diff
@@ -11,11 +11,11 @@ import (
 	"github.com/Azure/agentbaker/pkg/agent/datamodel"
 )
 
-// NBContractBuilder is a helper struct to build the NBContract (Node Bootstrap Contract).
-// It provides methods to apply configuration, get the NBContract object, and validate the contract, etc.
-type NBContractBuilder struct {
-	// nodeBootstrapConfig is the configuration object for the NBContract (Node Bootstrap Contract).
-	nodeBootstrapConfig *Configuration
+// AKSNodeConfigBuilder is a helper struct to build the AKSNodeConfig.
+// It provides methods to apply configuration, get the AKSNodeConfig object, and validate the contract, etc.
+type AKSNodeConfigBuilder struct {
+	// aKSNodeConfig is the configuration object for the AKSNodeConfig.
+	aKSNodeConfig *Configuration
```

Comment:
Thanks for the suggestion. 
I just checked what is the suggested way for camel case with leading acronym. It turns out most people suggest like yours `aks` instead of `aKS`. Updated all. Thanks.